ambry-api/src/main/java/com/github/ambry/config/CompressionConfig.java

```diff
@@ -21,8 +21,8 @@
 public class CompressionConfig {
 
   // Boolean whether compression is enabled in PUT operation.
-  static final String COMPRESSION_ENABLED = "router.compression.enabled";
-  static final boolean DEFAULT_COMPRESSION_ENABLED = true;
+  public static final String COMPRESSION_ENABLED = "router.compression.enabled";
+  static final boolean DEFAULT_COMPRESSION_ENABLED = false;
```

Comment:
why disable it if the bug is fixed ?

Reply:
we should make the default as false, so if we want to enable it, we have to turn it on in configuration file. In this way, we will be more aware of what is happening.